cocos/scripting/lua-bindings/auto/lua_cocos2dx_backend_auto.cpp

```diff
@@ -1998,24 +1998,21 @@ int lua_cocos2dx_backend_VertexLayout_setLayout(lua_State* tolua_S)
 #endif
 
     argc = lua_gettop(tolua_S)-1;
```

Comment:
Please revert the modification of this file, we wan to cocos robot modifies it automatically.